_includes/appe-appliance-console-cli.md

```diff
@@ -1,110 +1,98 @@
-| \--dbdisk        | database disk device path (for configuring an internal database)                           |
-| \--hostname (-h) | database hostname                                                                          |
-| \--port          | database port (defaults to `5432`)                                                         |
-| \--username (-U) | database username (defaults to `root`)                                                     |
-| \--password (-p) | database password                                                                          |
-| \--dbname (-d)   | database name (defaults to `vmdb_production`)                                              |
+| --region (-r)    | region number (create a new region in the database - requires database credentials passed) |
+| --internal (-i)  | internal database (create a database on the current appliance)                             |
+| --dbdisk         | database disk device path (for configuring an internal database)                           |
+| --hostname (-h)  | database hostname                                                                          |
+| --port           | database port (defaults to `5432`)                                                         |
+| --username (-U)  | database username (defaults to `root`)                                                     |
+| --password (-p)  | database password                                                                          |
+| --dbname (-d)    | database name (defaults to `vmdb_production`)                                              |
+
+{: caption="Table 1. Database Configuration Options" caption-side="top"}
 
-Database Configuration Options
+#### v2_key Options
 
 |                   |                                                            |
 | ----------------- | ---------------------------------------------------------- |
 | Option            | Description                                                |
-| \--key (-k)       | create a new v2\_key                                       |
-| \--fetch-key (-K) | fetch the v2\_key from the given host                      |
-| \--force-key (-f) | create or fetch the key even if one exists                 |
-| \--sshlogin       | ssh username for fetching the v2\_key (defaults to `root`) |
-| \--sshpassword    | ssh password for fetching the v2\_key                      |
+| --key (-k)        | create a new v2_key                                        |
+| --fetch-key (-K)  | fetch the v2_key from the given host                       |
+| --force-key (-f)  | create or fetch the key even if one exists                 |
+| --sshlogin        | ssh username for fetching the v2_key (defaults to `root`)  |
+| --sshpassword     | ssh password for fetching the v2_key                       |
 
-v2\_key Options
+{: caption="Table 2. v2_key Options" caption-side="top"}
```

Comment:
I think these captions don't work (or at least they likely come up as regular text.  We should clean up/remove these in a follow up PR.